vocabularies/Common.xml

```diff
@@ -124,6 +124,16 @@
         <Annotation Term="Core.RequiresType" String="Edm.DateTimeOffset" />
       </Term>
 
+      <Term Name="Locale" Type="Edm.String" Nullable="true" AppliesTo="Property Parameter ReturnType">
+        <Annotation Term="Common.Experimental" />
+        <Annotation Term="Core.Description" String="The value of the annotated property or parameter shall be interpreted in the given locale" />
+        <Annotation Term="Core.LongDescription">
+          <String>Locales shall be specified through a [Unicode Locale Identifier](https://www.unicode.org/reports/tr35/tr35.html#Identifiers).
+          If this annotation is absent or null, the current user's or default locale is used.
+          An example for the applicability of this annotation are calendar weeks, whose meaning is locale-dependent.</String>
```

Comment:
Don't you want to repair the regex for "calendar week" to also allow week 54?

Reply:
Yes, but for the fiscal week I would not want to specify any pattern, because I don't know how high that can go.

Reply:
Then let's remove the pattern altogether if we can't find out what the globally correct pattern is.

Reply:
Remove it also for calendar weeks, or only for fiscal weeks?

Reply:
For calendar weeks we would have to extend it to cover week 54 that can occur in the US, correct?